controllers/user.js

```diff
@@ -28,26 +27,33 @@ function index(req, res, next) {
 
 
 
-// //show function
-// function show(req, res) {
-//   Exercise.find({}, function(err, exercise) {
+//show function
+function show(req, res) {
+  
+//   Exercise.find({_id:req.params.id}, function(err, exercise) {
+//     console.log(exercise)
 //   res.render('user/index', { 
 //       exercise });
 //   });
-// };
+ };
 
 
 
   
 //create function
 function create(req, res) {
- const exercise = new Exercise(req.body)
-  exercise.save(function (err){
+ // console.log(create)
+ const log = new Log(req.body)
+console.log(log, "new log")
+ User.findOne({_id:req.user._id}, function(err, user) {
+//console.log(user)
+ user.exercises.push(log)
+  user.save(function (err){
     if(err)
      return res.send(err)
-    res.redirect("user/index")
+    res.redirect(`/user` )
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7841248)
```suggestion
    res.redirect("/user" )
```